### PR TITLE
Flavioarfaria/examples md update

### PR DESCRIPTION
**Pull request by flavioarfaria**
* Added Android Pay to the EXAMPLES.md file;
* Improved some outdated instructions (admin setup was using a deprecated process).

Please, don't hesitate to hunt for all sorts of English mistakes, or anything that could be better expressed in English. This is not my first language.

### CODE REVIEW

File: EXAMPLES.md [hunk 1]
```diff
@@ -292,15 +292,17 @@ ShopifyBuy.Client().products((products, error) => {
...
+To determine whether the user is able to make a payment with native pay, you can use the `CanCheckoutWithNativePay` method. If the user has this capability, then you can use `CheckoutWithNativePay` to present the native pay authentication interface to the user. If they do not, then you can accept payment using [Web View Checkout](#web-view-checkout).
 
 `CheckoutWithNativePay` takes in 4 parameters:
 
-* `key` is the Merchant ID of your application found on your [Apple Developer Portal](https://developer.apple.com/library/content/ApplePay_Guide/Configuration.html)
+* `key`
+  * **On iOS** is the Merchant ID of your application found on your [Apple Developer Portal](https://developer.apple.com/library/content/ApplePay_Guide/Configuration.html).
+  * **On Android** is a Base64-encoded public key found on _Admin_ > _Apps_ > _Manage private apps_ > _{Your private app}_ > _Storefront API_ > _Configure Mobile Buy SDK settings (optional)_ > _MOBILE BUY SDK (ANDROID)_.
```
Comment (sleroux) [inline EXAMPLES.md]: I would prefix the _Admin_->_Apps_ ... stuff saying that that is on the Shopify Admin site:

`**On Android** is a Base64-encoded public key that can be found on your Shopify Admin page. To locate the key, navigate to ...`

File: EXAMPLES.md [hunk 2]
```diff
@@ -349,13 +358,12 @@ To use Apple Pay, you must also enable **Background fetch** in the Player Settin
...
-
-To enable Apple Pay for your store through an app:
+### Enabling native pay on your Store
 
-1. Add the [Mobile App sales channel](https://help.shopify.com/api/sdks/custom-storefront/mobile-buy-sdk/add-mobile-app-sales-channel) in your Shopify admin.
-2. Enable Apple Pay in the Mobile App settings page.
+To enable native pay for your store through an app:
 
+1. Follow [these instructions](https://help.shopify.com/manual/apps/private-apps#generate-credentials-from-the-shopify-admin) to create a private app in your Shopify admin page.
+1. Enable _Allow this app to access your storefront data using the Storefront API_ flag in your private app configuration.
```
Comment (sleroux) [inline EXAMPLES.md]: Should be 2.

File: EXAMPLES.md [hunk 3]
```diff
@@ -370,13 +378,18 @@ On failure, you will receive a `ShopifyError`. There are 3 types of `ShopifyErro
 
 `HTTP` errors will be thrown when there was an issue connecting or downloading to a required web server.
 
-`NativePaymentProcessingError` will be thrown when Apple Pay fails to generate a token while trying to authenticate the user's card. This error is unrecoverable and you should fall back to a different payment method, or allow the user to try going through the process again.
+`NativePaymentProcessingError`
+
+This error is thrown whenever something goes wrong on the native pay side of the checkout. It is unrecoverable and you should fall back to a different payment method, or allow the user to try going through the process again. The reasons for this error to be thrown depend on the platform:
+
+* **iOS:** will be thrown when Apple Pay fails to generate a token while trying to authenticate the user's card.
+* **Android:** will be thrown for any unrecoverable Android Pay error that is prefixed with `ERROR_` ([see the full list](https://developers.google.com/android/reference/com/google/android/gms/wallet/WalletConstants)), such as _service unavailable_, _unsupported country_ or _authentication error_.
```
Comment (sleroux) [inline EXAMPLES.md]: I would reword the end of this sentence to say:

`You can see a full list of there errors [here](...)`

File: EXAMPLES.md (hunk 1 above)
Comment (flavioarfaria) [reply EXAMPLES.md]: https://github.com/Shopify/unity-buy-sdk/pull/362/commits/dde30a03aa1fc4bda959b69000b8a533647ebbbb

File: EXAMPLES.md (hunk 3 above)
Comment (flavioarfaria) [reply EXAMPLES.md]: https://github.com/Shopify/unity-buy-sdk/pull/362/commits/4a75989ae3c72b5698fd85a8b1ffdaedfc606f3d

File: EXAMPLES.md (hunk 2 above)
Comment (flavioarfaria) [reply EXAMPLES.md]: Actually, this was intentional because 1. is a valid markup for _whatever number this position must be in this enumeration_. I personally like to keep all of them 1s because if we delete, add or reorder items, we don't need to rewrite the numbers to keep them in order.

File: EXAMPLES.md [hunk 4]
```diff
@@ -319,11 +321,18 @@ ShopifyBuy.Client().products((products, error) => {
 
     cart.LineItems.AddOrUpdate(productVariantToCheckout, 1);
 
+    var key = null;
+    #if UNITY_IOS
+    key = "com.merchant.id";
+    #elif UNITY_ANDROID
+    key = "BL9QiRljozDhgfyfVHoK+l1l98fBY0x/in0rCYJxmTfnzJDWsX1+8l4HEa4LO0WeKQlYtuk8zcJtzimTMhr1UL8=";
```
Comment (richardmonette) [inline EXAMPLES.md]: Is this the same for all merchants, or will it have to be customized?

Comment (flavioarfaria) [reply EXAMPLES.md]: @richardmonette each merchant has its own, which is provided by the admin page. Instructions mention that. These two strings are placeholders for the sake of the example. Game devs will not use `com.merchant.id` for iOS either.
